doc/whats_new/upcoming_changes/sklearn.tree/30318.feature.rst

```diff
@@ -0,0 +1,5 @@
+- :class:`tree.ExtraTreeClassifier` and :class:`tree.ExtraTreeRegressor` now
+  support missing-values in the data matrix ``X``. Missing-values are handled by
+  randomly moving all of the samples to the left, or right child node as the tree is
+  traversed.
+  By :user:`Adam Li <adam2392>` and :user:`Loïc Estève <lesteve>`
```

Comment:
Since the feature has not been released I use the same content so that towncrier merges them and mention both PRs.

To be honest I would be fine not adding a changelog as well ...

Reply:
I don't mind. Maybe this is a nice implementation to have actually.